src/classic/clvm_tools/stages/stage_2/module.rs

```diff
@@ -489,7 +491,7 @@ fn compile_mod_stage_1(
                     }
                 }
 
-                let uncompiled_main = alist[alist.len() - 1];
+                let uncompiled_main = nonempty_last(allocator.null(), &alist)?;
```

Comment:
It's bad if this code panics (and the array index did, but the list had already been checked) because it's run from python.  When changing it i want it to be more conservative than how it was left.